### DISCUSSION TITLE
Recover Dashboard port :18083 password

### DISCUSSION THREAD

**Discussion by panjab2021** [emqx/emqx#7855] [Q&A] [answered]
Hi,

I installed new Emqx on Ubuntu , setup everything okay, logged into dashboard  : 18083 using default user and pass . I changed password and forget it now. I could not find any recovery option and need help to change this pass. 
Thanks

**Comment by advanisunny3** [comment, marked as answer]
I am not an expert but maybe try to reset admin password via command :
https://www.emqx.io/docs/en/v4.4/advanced/cli.html#admins-command

**Comment by panjab2021** [comment]
Thanks for your response and suggestion. I don't see the Bin folder on my
Ubuntu Linux server. It is installed under etc/emqx but there is no bin
folder. Please advise.
Thanks

**Reply by oxgen1** [reply]
```sudo emqx ctl admins passwd <Username> <password> ``` Worked for me
